lib/http-server.js

```diff
@@ -0,0 +1,154 @@
+var async = require('async-collection')
+var mkdirp = require('mkdirp')
+var http = require('http')
+var path = require('path')
+var pump = require('pump')
+var net = require('net')
+var fs = require('fs')
+var os = require('os')
+
+var selfsigned = require('selfsigned')
+var getPort = require('getPort')
```

Comment:
```
$ node ../../choojs/bankai/bin.js app.js
module.js:529
    throw err;
    ^

Error: Cannot find module 'getPort'
```

This require should be all lowercase `getport`

Reply:
Found that too, there is a typo there `getPort` -> `getport`

Reply:
Good catch; thanks!